ariadne/cardano/src/Ariadne/Wallet/Backend.hs

```diff
@@ -85,6 +89,19 @@ createWalletBackend walletConfig sendWalletEvent getPass voidPass = do
             sendWalletEvent $ WalletRequireConfirm mVar confirmType
             takeMVar mVar
 
+        walletFee'
+            :: WalletReference
+            -> [LocalAccountReference]
+            -> InputSelectionPolicy
+            -> NonEmpty TxOut
+            -> IO Coin
+        walletFee' walletRef accRefs _ txOuts = do
+            let payees = ((,) <$> txOutAddress <*> txOutValue) <$> txOuts
+            accountId <- ourAccountId pwl walletSelRef walletRef accRefs
```

Comment:
Well, it looks wrong, there can be more than one input account, so you should consider UTXOs of all of them.